src/Common/TenantDetailModel.cs

```diff
@@ -28,7 +27,7 @@ public class TenantDetailModel
         /// <summary>
         /// Gets or sets the tenant account type.
         /// </summary>
-        public ApplicationSubscriptionLevel SubscriptionLevel { get; set; }
+        //public ApplicationSubscriptionLevel SubscriptionLevel { get; set; }
```

Comment:
Is this property no longer needed?

Reply:
ok, now I see. The view here in github was strange. I have this marked as obsolete, so this commenting out is probably safe. Might want to double check Inspire so make sure this model isn't being referenced anymore.

Reply:
I checked and the TenantDetailModel isn't used in Inspire anywhere.